zebrad/tests/acceptance.rs

```diff
@@ -544,15 +544,18 @@ fn version_args() -> Result<()> {
     let testdir = testdir()?.with_config(&mut default_test_config()?)?;
     let testdir = &testdir;
 
-    // unexpected free argument `argument`
```

Comment:
You could use the tip-height command here instead of deleting this test. It doesn't expect any free arguments.

The default config and an empty test directory should be fine, because this test is about argument parsing.

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/6801/commits/6b553d290759d64255389c43c1dfb6698313b28b.